test/integration/csv.js

```diff
@@ -35,8 +37,39 @@ describe('bitcoinjs-lib (transactions w/ CSV)', function () {
     ])
   }
 
+  // 2 of 3 multisig of bQ, cQ, dQ, but after sequence1 time,
+  // aQ can allow the multisig to become 1 of 3.
+  // But after sequence2 time, aQ can sign for the output all by themself.
+  function complexCsvOutput (aQ, bQ, cQ, dQ, sequence1, sequence2) {
+    return bitcoin.script.compile([
+      bitcoin.opcodes.OP_IF,
+      bitcoin.opcodes.OP_IF,
+      bitcoin.opcodes.OP_2,
+      bitcoin.opcodes.OP_ELSE,
+      bitcoin.script.number.encode(sequence1),
+      bitcoin.opcodes.OP_CHECKSEQUENCEVERIFY,
+      bitcoin.opcodes.OP_DROP,
+      aQ.publicKey,
+      bitcoin.opcodes.OP_CHECKSIGVERIFY,
+      bitcoin.opcodes.OP_1,
+      bitcoin.opcodes.OP_ENDIF,
+      bQ.publicKey,
+      cQ.publicKey,
+      dQ.publicKey,
+      bitcoin.opcodes.OP_3,
+      bitcoin.opcodes.OP_CHECKMULTISIG,
+      bitcoin.opcodes.OP_ELSE,
+      bitcoin.script.number.encode(sequence2),
+      bitcoin.opcodes.OP_CHECKSEQUENCEVERIFY,
+      bitcoin.opcodes.OP_DROP,
+      aQ.publicKey,
+      bitcoin.opcodes.OP_CHECKSIG,
+      bitcoin.opcodes.OP_ENDIF
+    ])
```

Comment:
Indentation and readability for this is lame.
I wonder if we included a `String  -> Script` parser that simply tokenized the above via whitespace.  Meaning we could indent and group things?
No idea how you would deal with complex hex constants though..